### ISSUE TITLE
Incorrect "Oops" dialog when making container wider while holding Temperature constant.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/gas-properties#246]
Noticed during 5/202/24 design meeting.

Steps to reproduce:

1. Start the sim, go to _Ideal_ screen.
2. Add 50 heavy particles in Particles panel.
3. Select "Temperature (T)" radio button in Hold Constant panel.
4. Attempt to make the container wider by dragging the resize handle to the left (with either the mouse or keyboard). The dialog shown below opens. The lid is still on the container, pressure is nowhere near blowing the lid off, and making the container wider should not increase pressure.

<img width="734" alt="screenshot_3312" src="https://github.com/phetsims/gas-properties/assets/3046552/627682b8-6ccc-4fa2-8b7b-cc53826a3992">

**Comment by pixelzoom** [comment]
`lidIsOpenProperty` is derived in IdealGasLawContainer.ts:

```ts
    this.lidIsOpenProperty = new DerivedProperty(
      // this.widthProperty is used by this.getMaxLidWidth()
      [ this.lidIsOnProperty, this.lidWidthProperty, this.widthProperty ],
      ( lidIsOn, lidWidth, containerWidth ) => !lidIsOn || ( lidWidth < this.getMaxLidWidth() ), {
        tandem: options.tandem.createTandem( 'lidIsOpenProperty' ),
        phetioFeatured: true,
        phetioValueType: BooleanIO,
        phetioDocumentation: 'Whether the container\'s lid is open.'
      } );
```

Inspecting the dependencies:

lidIsOn => true
lidWidth => 8075
getMaxLidWidth() => 8082.876888736264

So the lid is on, but the lid's width does not cover the opening in the top of the container. 

I'm guessing that this is a problem with listener order. There's probably something that adjusts `lidWidthProperty` that has not executed yet.

**Comment by pixelzoom** [comment]
The problem is indicated by the comment here:

```ts
    this.lidIsOpenProperty = new DerivedProperty(
      // this.widthProperty is used by this.getMaxLidWidth()
      [ this.lidIsOnProperty, this.lidWidthProperty, this.widthProperty ],
```

`getMaxLidWidth` uses `widthProperty`.   So a dependency on `this.widthProperty` was added to appease axon's `strictAxonDependencies` option.  But `widthProperty` is updated before `lidWidthProperty`, and that results in an internmediate state where the container is slightly open. 

To resolve this problem, I removed the dependency on `widthProperty` and added:

 `strictAxonDependencies: false, // See https://github.com/phetsims/gas-properties/issues/246`

The problem is not longer reproducible, so closing.
